@xen-orchestra/lite/src/components/ui/UiCard.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="ui-card">
+  <div class="ui-card ui-card-backgroud">
```

Comment:
I was about to propose something like this too.

But instead of `hasError` prop, I would add a `color: Color` prop, like in some other components.

(Also, there is a type in `backgroud`)